tests/functional/test_receipt.py

```diff
@@ -113,6 +132,47 @@ def test_decode_logs_specify_abi_as_event(
     assert spy.call_count == 0
 
 
+def test_decode_logs_multiple_event_types(owner, contract_instance, assert_log_values):
```

Comment:
**note to reviewer**: i just reorganized some tests too so all tests with same prefix are by each other, makes it easier for me to add new ones